api/src/env.ts

```diff
@@ -177,6 +177,7 @@ const allowedEnvironmentVars = [
 	'EXPORT_BATCH_SIZE',
 	// flows
 	'FLOWS_EXEC_ALLOWED_MODULES',
+	'FLOWS_EXEC_ALLOWED_ENV',
```

Comment:
Now that it controls access to env vars for any operation, do we want to rename this to something like `FLOWS_ALLOWED_ENV` or `FLOWS_ENV_ALLOW_LIST`?

Reply:
Oh yes, definitely! Updated the changes. 👍